CMakeLists.txt

```diff
@@ -13,25 +13,24 @@ include(GNUInstallDirs)
 
 # Minimum Versions
 set(GLIB_MINIMUM_VERSION "2.50.0")
-set(LXQTBT_MINIMUM_VERSION "0.8.0")
-set(QT_MINIMUM_VERSION "5.12.0")
+set(LXQTBT_MINIMUM_VERSION "2.0.0")
+set(QT_MINIMUM_VERSION "6.6.0")
 
-set(OBCONFQT_MAJOR_VERSION 0)
-set(OBCONFQT_MINOR_VERSION 16)
-set(OBCONFQT_PATCH_VERSION 4)
+set(OBCONFQT_MAJOR_VERSION 1)
+set(OBCONFQT_MINOR_VERSION 0)
+set(OBCONFQT_PATCH_VERSION 0)
```

Comment:
Please only change the patch version to 5, such that it becomes 0.16.5. We don't want to pretend that we really maintain this app.